phpBB/language/en/ucp.php

```diff
@@ -411,7 +411,7 @@
 	'PASS_TYPE_SYMBOL_EXPLAIN'	=> 'Password must be between %1$s and %2$s long, must contain letters in mixed case, must contain numbers and must contain symbols.',
 	'PASSWORD'					=> 'Password',
 	'PASSWORD_ACTIVATED'		=> 'Your new password has been activated.',
-	'PASSWORD_UPDATED'			=> 'A new password was sent to your registered email address.',
+	'PASSWORD_UPDATED_IF_EXISTED'	=> 'If your account exists a new password was sent to your registered email address. If it does not, it may be because you are banned, not activated your account yet or not allowed to change password. Contact admin if that is the case.',
```

Comment:
```suggestion
	'PASSWORD_UPDATED_IF_EXISTED'	=> 'A new password has been sent to your email address if your login information is connected to an account. If you do not receive an email, it may be because you are banned, your account has not been activated yet, or you are not allowed to change passwords. Contact an admin if you continue to experience problems recovering access to your account.',
```

Reply:
@phpbb/native feedback pls? :)

Reply:
'PASSWORD_UPDATED_IF_EXISTED'	=> 'If your account exists, a new password was sent to your registered email address. If it does not, it may be because you are banned, your account is not activated, or you are not allowed to change your password. Contact admin if any of those reasons apply.',
 
	~~'PASSWORD_UPDATED_IF_EXISTED'	=> 'A new password has been sent to your registered email address if your login information is connected to an account. If you do not receive an email, it may be because you are banned, your account has not been activated yet, or you are not allowed to change passwords. Contact an admin if you continue to experience problems recovering access to your account.',~~